### ISSUE TITLE
rich presence time elapsed problem

### ISSUE THREAD

**Issue by XqMine** [devsnek/discord-rich-presence#35]
hi all ✋ , i have a problem with rich presence time
The problem : time elapsed keep repeating from 00:13 to 00:00 i think
screen shot:
![image](https://user-images.githubusercontent.com/89086974/168088406-ca1853b5-d482-4eda-940f-303ee0ebd469.png)
to
![image](https://user-images.githubusercontent.com/89086974/168088571-687570cd-0f54-4900-b9d7-5aa5c41cf199.png)
my code : 
![image](https://user-images.githubusercontent.com/89086974/168089364-d5bdb839-e7c7-4f62-944b-be9dc290f3ec.png)
i hope someone can helps me

**Comment by DeadLyBro** [comment]
Can you try this code; 
(it's from readme.md)

```
startTimestamp: Date.now(),
endTimestamp: Date.now() + 1337,        <--- its example for end time stamp.
```

Also idk your code is not work for me.

**Comment by ghost** [comment]
Is it because you keep updating your activity every 15 seconds?
```
setInterval(() => {
  setActivity();
}, 15 * 1000);
```

**Comment by DeadLyBro** [comment]
> Is it because you keep updating your activity every 15 seconds?
> 
> ```
> setInterval(() => {
>   setActivity();
> }, 15 * 1000);
> ```

Yeah he set activity every 15 sec so its repeating timer. U're right.
